tests/all/js_globals/Array.rs

```diff
@@ -40,6 +40,38 @@ fn index_of() {
         .test()
 }
 
+#[test]
+fn sort() {
+    project()
+        .file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js;
+
+            #[wasm_bindgen]
+            pub fn sort_array(this: &js::Array) -> js::Array {
+                this.sort()
+            }
+
+        "#)
+        .file("test.ts", r#"
+            import * as assert from "assert";
+            import * as wasm from "./out";
+
+            export function test() {
+                let numbers = [3, 1, 6, 2];
+                let sorted = wasm.sort_array(numbers);
+
+                for (let [index, number] of sorted.entries()) {
+                    assert.equal([1, 2, 3, 6][index], number);
```

Comment:
Nitpick: this assertion would be a little simpler with:

```js
assert.deepStrictEqual(sorted, [1, 2, 3, 6])
```

https://nodejs.org/api/assert.html#assert_assert_deepstrictequal_actual_expected_message